src/index.test.js

```diff
@@ -324,3 +324,57 @@ describe('time zone is carried over after a business-day operation', () => {
     expect(nyPlusTwo.zoneName).toBe('America/New_York');
   });
 });
+
+describe('diffBusiness()', () => {
+  it('knows two identical DateTimes have a business day diff of 0', () => {
+    const targetDate = DateTime.local().plus({ hours: 2 });
```

Comment:
Actually, this one with the hours might actually still need `.startOf` because the test could 2 hours into the next day. The other tests that add days should be okay.

Reply:
Agreed